### ISSUE TITLE
<title> has no effect

### ISSUE THREAD

**Issue by anton-johansson** [stoicflame/enunciate#243]
Hi!

I'm using the maven-enunciate-plugin, but the ```<title>``` tag seems to have no effect. I get the artifacts name in the POM as the title instead. The maven plugin sets the POM's name as the default title, as [ConfigMojo.java#L221](https://github.com/stoicflame/enunciate/blob/9cc94188ea63a29fcfbd50e5945d91ca4698c79d/slim-maven-plugin/src/main/java/com/webcohesion/enunciate/mojo/ConfigMojo.java#L221) clearly says, but looking at [EnunciateConfiguration.java#L63](https://github.com/stoicflame/enunciate/blob/45487fddb71577402057ec16e9ee0b07133608c7/core/src/main/java/com/webcohesion/enunciate/EnunciateConfiguration.java#L63) and [DocsModule.java#L246](https://github.com/stoicflame/enunciate/blob/4c490939062b06e398e642333d69b3866354a40c/docs/src/main/java/com/webcohesion/enunciate/modules/docs/DocsModule.java#L246), I don't quite understand why my overridden ```<title>``` won't work.

Any ideas?

Thanks!

**Comment by stoicflame** [comment]
I suspect that the title _is_ working, just not showing up where you'd expect.

The only place the title is used in the generated docs is in the `<title>` HTML tag and in the navigation menu. The text you see on the index page is the `<description>` configuration element.

![title-example](https://cloud.githubusercontent.com/assets/145838/12303492/e67c814c-b9e7-11e5-8552-5bf58d2a332b.png)

**Comment by anton-johansson** [comment]
Ahh, of course. My bad. I recall testing the description tag as well, but I'll give it another go first!

**Comment by anton-johansson** [comment]
~~Hmm, still can't get it to work. According to ```docs.fmt``` the title is used in two places. The ```<head><title>``` and the navbar. These are still my POM's ```<name>``` tag, no matter what I put in ```<title>```.~~

~~Then we we have to Jumbotron (the one that shows 'Enunciate Edge Case Example' in your picture), where I also get the POM's ```<name>``` tag. In ```docs.fmt``` this one reads the value from ```${apiDocs}```, which gets it value [from here](https://github.com/stoicflame/enunciate/blob/4c490939062b06e398e642333d69b3866354a40c/docs/src/main/java/com/webcohesion/enunciate/modules/docs/DocsModule.java#L235). This SHOULD read from the ```<description>``` tag, or by default take the POM's ```<description>``` tag. But I still get the POM's ```<name>``` tag here aswell.
Hmm, still can't get it to work. According to ```docs.fmt``` the title is used in two places. The ```<head><title>``` and the navbar. These are still my POM's ```<name>``` tag, no matter what I put in ```<title>```.~~

~~Then we we have to Jumbotron (the one that shows 'Enunciate Edge Case Example' in your picture), where I also get the POM's ```<name>``` tag. In ```docs.fmt``` this one reads the value from ```${apiDocs}```, which gets it value [from here](https://github.com/stoicflame/enunciate/blob/4c490939062b06e398e642333d69b3866354a40c/docs/src/main/java/com/webcohesion/enunciate/modules/docs/DocsModule.java#L235). This SHOULD read from the ```<description>``` tag, or by default take the POM's ```<description>``` tag. But I still get the POM's ```<name>``` tag here aswell.~~

OK, I figured it out. My bad, I didn't look hard enough in my console output. The output says:
```[INFO] [ENUNCIATE] Skipping documentation source generation as everything appears up-to-date...```
... even if I've changed the ```<title>``` or ```<description>``` tags.

Not sure if it's worth opening up an issue for, up to you. Otherwise I'll leave this. Thanks for the help!
